tests/test_jointcal_cfht.py

```diff
@@ -55,7 +55,8 @@ def test_jointcalTask_2_visits(self):
         # first run of jointcal on this data. We should always do better than
         # this in the future!
         relative_error = 25e-3*u.arcsecond
-        self._testJointCalTask(2, relative_error, absolute_error)
+        pa1 = 0.019
```

Comment:
Indeed, and that's a fine place for it. Now if I just knew what the SRD was, we'd be in business...